src/artemis/stores/mapgraph/read.cljs

```diff
@@ -154,7 +154,10 @@
                 (reduced ::incomplete-value)))
 
             (map? expr)
-            (pull-join store result expr entity gql-context)
+            (let [map-result (pull-join store result expr entity gql-context)]
+              (if (= map-result ::incomplete-value)
```

Comment:
could you use the `incomplete?` function here?

Reply:
yep